org.schabi.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>org.schabi.NewPipe</id>
+  <name>NewPipe (experimental Flatpak version)</name>
+  <summary>A free lightweight YouTube frontend for Android.</summary>
+  <project_license>GPL-3.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="vcs-browser">https://github.com/TeamNewPipe/NewPipe</url>
+  <url type="homepage">https://newpipe.net/</url>
```

Comment:
then the app-id should be `net.newpipe.NewPipe`

Reply:
done